.circleci/config.yml

```diff
@@ -33,32 +33,41 @@ jobs:
           name: Run Tests
           command: |
             . venv/bin/activate
+            mypy sprints
             coverage run -m pytest
             coverage report
+
+  deploy:
+    docker:
+      - image: circleci/python:3.7
+    steps:
       - setup_remote_docker:
           docker_layer_caching: true
       - run:
           name: Install Docker Compose
-          filters:
-            branches:
-              only: master
           command: |
             curl -L https://github.com/docker/compose/releases/download/1.19.0/docker-compose-`uname -s`-`uname -m` > ~/docker-compose
             chmod +x ~/docker-compose
             sudo mv ~/docker-compose /usr/local/bin/docker-compose
       - run:
           name: Build and push Docker image
-          filters:
-            branches:
-              only: master
           command: |
             echo $DOCKER_PASSWORD | docker login -u $DOCKER_LOGIN --password-stdin
             docker-compose -f production.yml build
             docker-compose -f production.yml push
```

Comment:
@Agrendalath, I don't know a lot about docker - does this push the built image(s) to a docker registry? If so, which registry is it - Docker Hub? Do the images contain any secrets baked in? If so, we cannot push the image to a public registry.

Reply:
The images don't contain any secrets. They are loaded from `.env` files.